src/index.js

```diff
@@ -36,6 +36,11 @@ const schema = {
         "Validate generated `*.d.ts` files and fail if an update is needed (useful in CI). Defaults to `false`",
       type: "boolean",
     },
+    configFile: {
```

Comment:
I'd suggest calling it `prettierConfigFile` to make it clear that it has nothing to do with the loader config. Or perhaps extending `formatter` to support objects, eg:
```js
formatter: {
  type: 'prettier',
  configFile: './.prettierrc'
}
```

Reply:
Okay, im rename it and update pull request soon

Reply:
Thanks! Really sorry about the delay, I added the comment ages ago but forgot to submit the review.

Reply:
@Obi-Dann done